recipes/dmox/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.1.1" %}
+{% set sha256 = "d08efd4fc59be9a481d24e3f6ae611a343e6d03fa49552c898901ba3b55896de" %}
+
+package:
+  name: dmox
+  version: '{{ version }}'
+
+source:
+  url: https://gitlab.mbb.cnrs.fr/ibonnici/dmox/-/archive/v{{ version }}/dmox-v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('dmox', max_pin="x") }}
+  script: |
+    if [ `uname` == Darwin ]; then
+      export HOME=`mktemp -d`
+    fi
+    cargo test --release
+    cargo install -v --locked --no-track --root $PREFIX --path .
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }}
```

Comment:
Other Rust recipes use cargo-bundle-license tool to collect the third party licences too. 
Please check the docs or grep for 'cargo-bundle-license' to see an example.

Reply:
Interesting, thanks, I'll try that.

Reply:
Done in https://github.com/bioconda/bioconda-recipes/pull/54435/commits/86fee04e3462ee2382a4545882f3d17b2b42604a.